pkg/infoschema/test/clustertablestest/tables_test.go

```diff
@@ -647,7 +647,10 @@ func TestSystemSchemaID(t *testing.T) {
 	_, dom := testkit.CreateMockStoreAndDomain(t)
 
 	uniqueIDMap := make(map[int64]string)
-	checkSystemSchemaTableID(t, dom, "information_schema", autoid.InformationSchemaDBID, 1, 10000, uniqueIDMap)
+	// [5000, 10000) of information_schema is reserved for PingKaiDB. See
```

Comment:
'PingKaiDB' is a private project in PingCAP...which is confusing to the community.

I think it's better to say something like '... is reserved for the downstream projects'